test/inductor/test_aot_inductor_package.py

```diff
@@ -515,10 +513,9 @@ def default(*args, **kwargs):
                         )
 
     @unittest.skipIf(
-        _get_torch_cuda_version() < (12, 6), "Test is only supported on CUDA 12.6+"
+        torch.version.hip is None and _get_torch_cuda_version() < (12, 6), "Test is only supported on CUDA 12.6+"
```

Comment:
Just wondering why we need this change, seems that skipping if torch.version.hip is None is same as skipping on NV. We should just need to remove the skipIfRocm line IIUC.